checks.lua

```diff
@@ -226,17 +234,21 @@ function checkers.int64(arg)
     return false
 end
 
-checkers.tuple = box.tuple.is
+if is_tarantool() == true then
+    checkers.tuple = box.tuple.is
+end
```

Comment:
I suggest to move all Tarantool-specific stuff to single place

Reply:
Done.